src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineNameModal/EditBaselineNameModal.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Modal, TextInput } from '@patternfly/react-core';
+
+import { editNameModalActions } from './redux';
+import { baselinesTableActions } from '../../../BaselinesTable/redux';
+
+class EditBaselineNameModal extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            baselineId: this.props.baselineId,
+            baselineName: this.props.baselineName
+        };
+
+        this.updateBaselineName = (value) => {
+            this.setState({ baselineName: value });
+        };
+    }
+
+    confirmModal = () => {
+        const { baselineId, baselineName } = this.state;
+        const { patchBaseline, toggleEditNameModal } = this.props;
+
+        /*eslint-disable camelcase*/
+        patchBaseline(baselineId, { display_name: baselineName, facts_patch: []});
+        /*eslint-enable camelcase*/
+
+        toggleEditNameModal();
+    }
+
+    cancelModal() {
+
+    }
```

Comment:
You shall not cancel!

Reply:
Maybe a comment in the empty function to make it more obvious why it needs to exist.